src/proposals/20251113/Spell_20251113.t.sol

```diff
@@ -0,0 +1,412 @@
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseVariableBorrowRate:        uint256(ssrRate) + newParams.baseRateSpread,
+                variableRateSlope1:            newParams.variableRateSlope1,
+                variableRateSlope2:            newParams.variableRateSlope2
+            })
+        );
+
+        assertEq(ITargetBaseIRMLike(configAfter.interestRateStrategy).getBaseVariableBorrowRateSpread(), newParams.baseRateSpread);
+
+        address expectedIRM = address(new RateTargetBaseInterestRateStrategy(
+            IPoolAddressesProvider(address(ctx.poolAddressesProvider)),
+            ICustomIRMLike(newParams.irm).RATE_SOURCE(),
+            newParams.optimalUsageRatio,
+            newParams.baseRateSpread,
+            newParams.variableRateSlope1,
+            newParams.variableRateSlope2
+        ));
+
+        _assertBytecodeMatches(expectedIRM, newParams.irm);
+    }
+
+    function test_ETHEREUM_sparkLend_withdrawUsdsDaiReserves() external onChain(ChainIdUtils.Ethereum()) {
+        uint256 spDaiBalanceBefore  = IERC20(Ethereum.DAI_SPTOKEN).balanceOf(Ethereum.ALM_PROXY);
+        uint256 spUsdsBalanceBefore = IERC20(Ethereum.USDS_SPTOKEN).balanceOf(Ethereum.ALM_PROXY);
+
+        assertEq(spDaiBalanceBefore,  413_852_826.907635248963156256e18);
+        assertEq(spUsdsBalanceBefore, 171_338_343.119011364190523468e18);
+
+        _executeAllPayloadsAndBridges();
+
+        assertEq(IERC20(Ethereum.DAI_SPTOKEN).balanceOf(Ethereum.DAI_TREASURY), 0);
+        assertEq(IERC20(Ethereum.USDS_SPTOKEN).balanceOf(Ethereum.TREASURY),    0);
+        assertEq(IERC20(Ethereum.DAI_SPTOKEN).balanceOf(Ethereum.ALM_PROXY),    spDaiBalanceBefore + 89_733.707492554224916353e18);
+        assertEq(IERC20(Ethereum.USDS_SPTOKEN).balanceOf(Ethereum.ALM_PROXY),   spUsdsBalanceBefore + 43_003.135094118145514323e18);
+    }
+
+    function test_ETHEREUM_sparkLend_depreciateSUSDSandSDAI() public onChain(ChainIdUtils.Ethereum()) {
+        SparkLendContext memory ctx = _getSparkLendContext();
+
+        ReserveConfig[] memory allConfigsBefore = _createConfigurationSnapshot('', ctx.pool);
+
+        ReserveConfig memory susdsConfigBefore = _findReserveConfigBySymbol(allConfigsBefore, 'sUSDS');
+        ReserveConfig memory sdaiConfigBefore  = _findReserveConfigBySymbol(allConfigsBefore, 'sDAI');
+
+        _executeAllPayloadsAndBridges();
+
+        ReserveConfig[] memory allConfigsAfter = _createConfigurationSnapshot('', ctx.pool);
+
+        // SUSDS
+
+        assertEq(susdsConfigBefore.ltv,                  79_00);
+        assertEq(susdsConfigBefore.liquidationThreshold, 80_00);
+        assertEq(susdsConfigBefore.liquidationBonus,     105_00);
+
+        ReserveConfig memory susdsConfigAfter = susdsConfigBefore;
+
+        susdsConfigAfter.ltv = 0;
+
+        _validateReserveConfig(susdsConfigAfter, allConfigsAfter);
+
+        _assertSupplyCapConfig({
```

Comment:
Do the same test as the last spell, assert the new cap amount and then assert that supply caps can't be changed with the automator anymore